go.mod

```diff
@@ -1,40 +1,41 @@
 module github.com/open-policy-agent/cert-controller
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.24.1
```

Comment:
whats the thing that requires this bump?

Reply:
`toolchain` was not needed, removed it.

Reply:
it was updated with my local go version.